Arrowgene.Ddon.Shared/Model/Storages.cs

```diff
@@ -68,6 +68,17 @@ public void Clear()
             }
         }
 
+        public void Clear(List<StorageType> storageTypes)
```

Comment:
Is this being used?

Reply:
Initially I was using it, but then I decided to use it differently. That said, it is just an overload to an existing function.